server/chat.ts

```diff
@@ -974,7 +974,7 @@ export class CommandContext extends MessageContext {
 			user.lastMessageTime = Date.now();
 		}
 
-		if (room && room.highTraffic &&
+		if (room?.highTraffic &&
 			toID(message).replace(/[^a-z]+/, '').length < 2
 			&& !user.can('broadcast', null, room)) {
```

Comment:
just while i'm here, what's the standard for where to put the &&?

Reply:
End of line.

Reply:
`.` is the only infix operator that should be at the beginning of a line.

Reply:
https://eslint.org/docs/rules/operator-linebreak

Oh, good, it's eslint-enforceable.